fccn/frontend-component-footer-nau
# feat(redwood): add nau learning footer customization

This PR migrates the custom NAU footer changes to the Redwood version. 

![image](https://github.com/fccn/frontend-component-footer-nau/assets/66016493/1c701109-1169-45f8-a720-2229e8ce97a9)

**How to test**

1. Create a tutor redwood environment.
2.  Inside your environment clone the learning MFE and mount it with `tutor mounts add ./frontend-app-learning`.
3. Inside the learning MFE, clone the footer repository with the current branch (remember to run `npm install` for both, the learning app and the footer).
4. Create the `module.config.js` file and add the following configuration.
```javascript
module.exports = {
  localModules: [
    { moduleName: '@edx/frontend-component-footer/dist', dir: './frontend-component-footer-nau', dist: 'src' },
    { moduleName: '@edx/frontend-component-footer', dir: './frontend-component-footer-nau', dist: 'src' },
  ],
};
```
3. Run the command `tutor images build learning-dev`.
4. Start your environment `tutor dev start`.
5. Create a course and configure the organization info inside the LMS admin.
6. Open the courseware and check you can see the changes.
